deploy-cloudgov-docker.sh

```diff
@@ -1,30 +1,44 @@
 #!/bin/sh
 #
-# This script will attempt to create the services required
-# and then launch everything.
-#
 
-# The deployment strategy you wish to employ ( rolling update or setting up a new environment)
+# WHAT THIS DOES
+
+# This script creates and launches the Cloud.gov services required to run the TDP app.
+
+# DEPENDENCIES
+
+# This script assumes that the user is logged in to Cloud.gov via the Cloud Foundry CLI,
+# and has permissions to provision services in the relevant Cloud.gov spaces and orgs.
+# For more, see: https://cloud.gov/docs/getting-started/setup/.
+
+# ARGUMENTS
+
+# The deployment strategy you wish to employ.
+# One of "rolling" (rolling update; deploy) or "setup" (setting up a new environment).
 DEPLOY_STRATEGY=${1}
 
-# The environment in which you want to execute these commands
+# The environment in which you want to execute these commands.
+# If set to "prod", provisions medium-psql-redundant in Cloud.gov instead of shared-psql.
+# For more, see: https://cloud.gov/docs/services/relational-database/.
+# (TODO: Consider renaming this argument to better reflect what it does.)
```

Comment:
I'm thinking of these TODOs as future TODOs and not TODOs for this PR, since my goal here is to move incrementally as we update and improve the deploy script.